lte/gateway/python/integ_tests/common/service303_utils.py

```diff
@@ -329,11 +327,9 @@ def _verify_metric_differences(
         diff = final_vals[i].value - initial_vals[i].value
         expected = expected_vals[i].value
         logging.debug(
-            'Metric %s %s increased by %d, expected %d',
-            expected_vals[i].name,
-            str(expected_vals[i].labels),
-            diff,
-            expected,
+            f'Metric {expected_vals[i].name} \
+            {str(expected_vals[i].labels)} increased by {diff}, \
+            expected {expected}',
```

Comment:
I think this adds breaks in that are different to the original, since the new code here gives something like:
```
DEBUG:root:Metric test             test increased by 1,             expected 1
```
when I run a test
```
import logging
logging.basicConfig(level=logging.DEBUG)

s = "test"
i = 1

logging.debug(
            f'Metric {s} \
            {s} increased by {i}, \
            expected {i}',
)
```

Perhaps you want something more like:
```
logging.debug(
            f"Metric {s} "
            f"{s} increased by {i}, "
            f"expected {i}"
)
```
?